RecoTracker/MkFit/plugins/MkFitIterationConfigESProducer.cc

```diff
@@ -82,6 +82,115 @@ namespace {
       part.m_sort_score[i] = maxEta_regSort * (reg - TrackerInfo::Reg_Barrel) + eta;
     }
   }
+
+  [[maybe_unused]] void partitionSeeds1(const TrackerInfo &trk_info,
+                                        const TrackVec &in_seeds,
+                                        const EventOfHits &eoh,
+                                        IterationSeedPartition &part) {
+    const LayerInfo &tib1 = trk_info.m_layers[4];
+    const LayerInfo &tob1 = trk_info.m_layers[10];
+
+    const LayerInfo &tidp1 = trk_info.m_layers[21];
+    const LayerInfo &tidn1 = trk_info.m_layers[48];
+
+    const LayerInfo &tecp1 = trk_info.m_layers[27];
+    const LayerInfo &tecn1 = trk_info.m_layers[54];
+
+    // Merge first two layers to account for mono/stereo coverage.
+    // TrackerInfo could hold joint limits for sub-detectors.
+    const auto &L = trk_info.m_layers;
+    const float tidp_rin = std::min(L[21].m_rin, L[22].m_rin);
+    const float tidp_rout = std::max(L[21].m_rout, L[22].m_rout);
+    const float tecp_rin = std::min(L[27].m_rin, L[28].m_rin);
+    const float tecp_rout = std::max(L[27].m_rout, L[28].m_rout);
+    const float tidn_rin = std::min(L[48].m_rin, L[49].m_rin);
+    const float tidn_rout = std::max(L[48].m_rout, L[49].m_rout);
+    const float tecn_rin = std::min(L[54].m_rin, L[55].m_rin);
+    const float tecn_rout = std::max(L[54].m_rout, L[55].m_rout);
```

Comment:
During the review of the Patatrack PRs we received *a lot* of comments regarding the use of "magic" numbers in the pixel code - even though most of the time we had simply modified or copied existing code.

Could you fix throughout this (and similar files, if any) the use of `4`, `10`, `21`, `48`, `27`, `54`, ... ?

Reply:
@fwyzard, thanks for your comment. This has been addressed in #36246.